account_invoice_cutoff_policy/models/account_invoice.py

```diff
@@ -17,12 +16,12 @@ def _get_invoice_partner(self):
         return self.commercial_partner_id
 
     @api.onchange("payment_term_id", "date_invoice")
-    def _onchange_payment_term_date_invoice(self):
+    def _onchange_invoice_date(self):
```

Comment:
No this is wrong.  For a start there isn't even a field called date_invoice anymore. But also it doesn't seem to point to the same places as the original function.

Reply:
Also we are now going to have extra fields to take care of like date for example.